drivers/nuopc/ocn_import_export.F90

```diff
@@ -581,6 +582,14 @@ subroutine ocn_import( importState, flds_scalar_name, ldiag_cpl, errorCode, rc )
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
     VSTOKES(:,:,:) = work1(:,:,:) * RCALCT(:,:,:) ! Stokes drift (m/s)
 
+    call state_getimport(importState, 'Sw_hstokes', work1, rc=rc)
+    if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    where (IFRAC <= 0.05_r8)
+       LASL(:,:,:) = work1 * RCALCT(:,:,:) ! surface layer Lanmguir number (unitless)
+    elsewhere
+       LASL(:,:,:) = c10000 ! a big number
```

Comment:
Same as above, you may want to consider `bignum` instead of `c10000`

Reply:
Done